webapp/channels/src/components/widgets/modals/components/save_changes_panel.scss

```diff
@@ -124,5 +125,10 @@
             background-color: var(--button-color);
             color: var(--denim-button-bg);
         }
+
+        &:disabled {
+            background: #FFF;
```

Comment:
Does this colour need to be themeable? (And is that spelled themeable or themable? 😛)

Reply:
done